routing.rst

```diff
@@ -243,6 +296,30 @@ arbitrary matching logic:
 
 .. configuration-block::
 
+    .. code-block:: php-attributes
+
+        // src/Controller/DefaultController.php
+        namespace App\Controller;
+
+        use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+        use Symfony\Component\Routing\Annotation\Route;
+
+        class DefaultController extends AbstractController
+        {
+            #[Route(
+                '/contact',
+                name: 'contact',
+                condition: "context.getMethod() in ['GET', 'HEAD'] and request.headers.get('User-Agent') matches '/firefox/i'",
+            )]
+            // expressions can also include config parameters:
+            // condition: "request.headers.get('User-Agent') matches '%app.allowed_browsers%'"
```

Comment:
Does this work (not having the attributes directly attached to the method they belong to)?

Reply:
Yes. It is directly attached. There's just whitespaces and comments between the attribute block and the method.

Reply:
Example: https://3v4l.org/Xn6oU (Note that 3v4l.org does not support the final attribute syntax yet)